tracer/test/test-applications/regression/Sandbox.AutomaticInstrumentation/Program.cs

```diff
@@ -0,0 +1,105 @@
+using Samples;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Security;
+using System.Security.Permissions;
+using System.Text;
+using System.Threading;
+
+namespace Sandbox.AutomaticInstrumentation
+{
+    public static class Program
+    {
+        public static int Main(string[] args)
+        {
+            // Set up local web server
+            string port = args.FirstOrDefault(arg => arg.StartsWith("Port="))?.Split('=')[1] ?? "9000";
+            using var server = WebServer.Start(port, out var url);
+            server.RequestHandler = HandleHttpRequests;
+            Console.WriteLine();
+            Console.WriteLine($"Starting HTTP listener at {url}");
+            Console.WriteLine();
+
+            // Set the minimum permissions needed to run code in the new AppDomain
+            PermissionSet permSet = new PermissionSet(PermissionState.None);
+            permSet.AddPermission(new SecurityPermission(SecurityPermissionFlag.Execution)); // REQUIRED to run code.
+            permSet.AddPermission(new SecurityPermission(SecurityPermissionFlag.UnmanagedCode)); // REQUIRED for automatic instrumentation.
+            // permSet.AddPermission(new WebPermission(PermissionState.Unrestricted)); // REQUIRED for application to send traces to the Agent over HTTP. Also enables application to generate automatic instrumentation spans.
```

Comment:
Should this be uncommented if it's required?

Reply:
Absolutely, that was a miss on my part. Fixed in a50a892fba823f06a26c3680ab973a63c244ca43